hadoop-ozone/dist/src/main/compose/ozonesecure/docker-compose.yaml

```
@@ -42,6 +41,8 @@ services:
     command: ["/opt/hadoop/bin/ozone","datanode"]
     env_file:
       - docker-config
+    environment:
+      KERBEROS_KEYTABS: dn
```

Seems like acceptance test failure is caused by missing `HTTP` keytab on datanode.

```
datanode_1  | 2020-01-17 12:30:34,899 [main] ERROR ozone.HddsDatanodeService: HttpServer failed to start.
...
datanode_1  | Caused by: javax.servlet.ServletException: javax.servlet.ServletException: Keytab does not exist: /etc/security/keytabs/HTTP.keytab
```

```suggestion
      KERBEROS_KEYTABS: dn HTTP
```